go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/k8snetworkplumbingwg/kubemacpool
 
-go 1.19
+go 1.21.9
```

Comment:
Thanks, did you run make vendor ?
asking because iirc a `toolchain` would be added once you do
but on the other hand i guess you did

Reply:
did not. will do now

Reply:
FYI I ran it now, but the `toolchain` thing was not added

Reply:
maybe ye, anyhow once you ran we are good
(worth to add make vendor on CI unit tests on a followup)

Reply:
btw seems it is from 1.21
https://go.dev/doc/toolchain
not sure why it is not added, but we will figure in the future, doesn't matter as long as everything works